pkg/sdk/poc/generator/list_builders.go

```diff
@@ -0,0 +1,6 @@
+package generator
+
+func (v *QueryStruct) List(name string, itemKind string, transformer FieldTransformer) *QueryStruct {
```

Comment:
hmmm, in my change there is a different `List` added, so we should either merge them or change their names respectively (https://github.com/Snowflake-Labs/terraform-provider-snowflake/pull/2171/files#diff-7d0e3fbb8967ad1b37eea35ab269bebbeb403363ebf75f5c43fdb10e77d7d839R65)

Reply:
I needed one that would be more flexible and accept FieldTransformer interface. so List could represent a Go slice and still be a keyword or parameter, which I think was my case, so I think we can use that one and adjust the one in your pr. wdyt ?

Reply:
Yep, we can leave yours and I will rename mine to be more specific